inst/@sym/isempty.m

```diff
@@ -60,7 +60,6 @@
 %! a = sym ([1 2]);
 %!assert (~isempty (sym (1)))
 %!assert (isempty (sym (se)))
-%!assert (isempty (se == []))
```

Comment:
elementwise comparision I suppose.  But even if the test is wrong, its not this PRs position to change it.  If the test is wrong, do another PR.

Unless this PR changes behaviour in some way that the test used to pass and now it fails...  But if so I'm worried because this PR is just a code refactor.